includes/Core/Authentication/Authentication.php

```diff
@@ -279,6 +279,11 @@ public function register() {
 		add_filter( 'googlesitekit_setup_data', $this->get_method_proxy( 'inline_js_setup_data' ) );
 		add_filter( 'googlesitekit_is_feature_enabled', $this->get_method_proxy( 'filter_features_via_proxy' ), 10, 2 );
 
+		add_action( 'get_transient_features', $this->get_method_proxy( 'get_transient_features' ) );
+		if ( ! wp_next_scheduled( 'get_transient_features' ) && ! wp_installing() ) {
```

Comment:
Nice one for catching `wp_next_scheduled` and `wp_installing` here, which weren't detailed in the IB :+1: